addons/website_forum/tests/test_forum_post.py

```diff
@@ -0,0 +1,38 @@
+from odoo.addons.website_forum.tests.common import TestForumCommon
+
+
+class TestForumPost(TestForumCommon):
+
+    def forum_tags(self, user, forum, size=10):
```

Comment:
Could move in common ? And would make it private, like ``_create_forum_tags(...)`